docs/articles/baseplatform/deploying-image.rst

```diff
@@ -6,6 +6,33 @@ Deploying the image
 This section describes how to deploy the build output, a ready disk-image, to a removable media.
 Further information can be found in the `Poky documentation`_.
 
+Recommended instructions
```

Comment:
why?

Reply:
Its faster than dd-ing?

Reply:
are you asking me?

Reply:
Im telling you.. its more than twice faster.

Reply:
Tell this to a reader.